### ISSUE TITLE
ModMenu no more works under 22w43a or beyond

### ISSUE THREAD

**Issue by ghost** [Prospector/ModMenu#494]
ModMenu no more works under 22w43a or beyond. 

```
java.lang.IllegalAccessError: class com.terraformersmc.modmenu.event.ModMenuEventHandler tried to access private field net.minecraft.class_339.field_22761 (com.terraformersmc.modmenu.event.ModMenuEventHandler and net.minecraft.class_339 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @c038203)
	at com.terraformersmc.modmenu.event.ModMenuEventHandler.shiftButtons(ModMenuEventHandler.java:146)
	at com.terraformersmc.modmenu.event.ModMenuEventHandler.afterTitleScreenInit(ModMenuEventHandler.java:61)
	at com.terraformersmc.modmenu.event.ModMenuEventHandler.afterScreenInit(ModMenuEventHandler.java:45)
	at net.fabricmc.fabric.api.client.screen.v1.ScreenEvents.lambda$static$2(ScreenEvents.java:103)
	at net.minecraft.class_437.handler$zih000$afterInitScreen(class_437.java:1824)
	at net.minecraft.class_437.method_25423(class_437.java:384)
	at net.minecraft.class_310.method_1507(class_310.java:1036)
	at net.minecraft.class_310.<init>(class_310.java:667)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.terraformersmc.modmenu.event.ModMenuEventHandler.shiftButtons(ModMenuEventHandler.java:146)
	at com.terraformersmc.modmenu.event.ModMenuEventHandler.afterTitleScreenInit(ModMenuEventHandler.java:61)
	at com.terraformersmc.modmenu.event.ModMenuEventHandler.afterScreenInit(ModMenuEventHandler.java:45)
	at net.fabricmc.fabric.api.client.screen.v1.ScreenEvents.lambda$static$2(ScreenEvents.java:103)
	at net.minecraft.class_437.handler$zih000$afterInitScreen(class_437.java:1824)
	at net.minecraft.class_437.method_25423(class_437.java:384)
	at net.minecraft.class_310.method_1507(class_310.java:1036)
	at net.minecraft.class_310.<init>(class_310.java:667)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		CoreMessaging.dll:Microsoft CoreMessaging Dll:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		DEVOBJ.dll:Device Information Set DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		GLU32.dll:OpenGL Utility Library DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		MMDevApi.dll:MMDevice API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		MSCTF.dll:MSCTF Server DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.2209.7 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		OpenAL.dll:Main implementation library:1.21.1:
		PSAPI.DLL:Process Status Helper:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		SETUPAPI.dll:Windows Setup API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.28.29913.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WINSTA.dll:Winstation Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WINTRUST.dll:Microsoft Trust Verification APIs:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		WTSAPI32.dll:Windows Remote Desktop Session Host Server SDK APIs:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		apphelp.dll:Application Compatibility Client Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		cfgmgr32.dll:Configuration Manager DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		cryptnet.dll:Crypto Network Related API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:DHCP Client Service:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6 Client:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		dinput8.dll:Microsoft DirectInput:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		drvstore.dll:Driver Store API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		dwmapi.dll:Microsoft Desktop Window Manager API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		hook64.dll
		icm32.dll:Microsoft Color Management Module (CMM):10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		inputhost.dll:InputHost:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		javaw.exe:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		jemalloc.dll
		jimage.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		jli.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		jna14968142455363719072.dll:JNA native library:6.1.4:Java(TM) Native Access (JNA)
		jsvml.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		jvm.dll:Java HotSpot(TM) 64-Bit server VM:17.0.5.0:Oracle Corporation
		kernel.appcore.dll:AppModel API Host:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		management.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		management_ext.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		msasn1.dll:ASN.1 Runtime APIs:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		mscms.dll:Microsoft Color Matching System DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		msvcp140.dll:Microsoft® C Runtime Library:14.28.29913.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		nio.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:NT Layer DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		ntmarta.dll:Windows NT MARTA provider:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		nvoglv64.dll:NVIDIA Compatible OpenGL ICD:31.0.15.1757:NVIDIA Corporation
		nvspcap64.dll:NVIDIA Game Proxy:3.25.1.27:NVIDIA Corporation
		opengl32.dll:OpenGL Client DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		powrprof.dll:Power Profile Helper DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		textinputframework.dll:"TextInputFramework.DYNLINK":10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		uxtheme.dll:Microsoft UxTheme Library:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.28.29913.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
		win32u.dll:Win32u:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		windows.staterepositorycore.dll:Windows StateRepository API Core:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		wldp.dll:Windows Lockdown Policy:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		xinput1_4.dll:Microsoft Common Controller API:10.0.25227.1000 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:Java(TM) Platform SE binary:17.0.5.0:Oracle Corporation
Stacktrace:
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
```

**Comment by jackassmc** [comment]
Blocked by https://github.com/FabricMC/yarn/pull/3367 (`getY` and `setY`).

**Comment by ghost** [comment]
> Blocked by [FabricMC/yarn#3367](https://github.com/FabricMC/yarn/pull/3367) (`getY` and `setY`).

Another PR: https://github.com/FabricMC/yarn/pull/3371 and https://github.com/FabricMC/yarn/pull/3369

**Comment by jackassmc** [comment]
The relevant PR is https://github.com/FabricMC/yarn/pull/3371 and it might get delayed until next snapshot.

**Comment by Bawnorton** [comment]
Made a PR, yarn was updated this morning #496

**Comment by ghost** [comment]
I tried raw version, but in-game I not found `Mods` button. Have only from main menu.

**Comment by Bawnorton** [comment]
I updated the PR. Missed that.

**Comment by jackassmc** [comment]
Fixed in https://github.com/TerraformersMC/ModMenu/pull/496.
